packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java

```diff
@@ -244,68 +211,38 @@ public ResponseEntity<UUID> equationsToModel(
 			}
 		}
 
-		// Get an AMR from Skema Unified Service or MIRA
 		final Model responseAMR;
-		String extractionService = "mira";
-		if (req.get("extractionService") != null) {
-			extractionService = req.get("extractionService").asText();
-		}
+		final TaskRequest latexToSympyRequest;
+		final TaskResponse latexToSympyResponse;
+		final TaskRequest sympyToAMRRequest;
+		final TaskResponse sympyToAMRResponse;
 
-		if (extractionService.equals("mira")) {
-			final TaskRequest latexToSympyRequest;
-			final TaskResponse latexToSympyResponse;
-			final TaskRequest sympyToAMRRequest;
-			final TaskResponse sympyToAMRResponse;
+		try {
+			// 1. LaTeX to sympy code
+			latexToSympyRequest = createLatexToSympyTask(equationsReq);
+			latexToSympyResponse = taskService.runTaskSync(latexToSympyRequest);
 
-			try {
-				// 1. LaTeX to sympy code
-				latexToSympyRequest = createLatexToSympyTask(equationsReq);
-				latexToSympyResponse = taskService.runTaskSync(latexToSympyRequest);
-
-				// 2. hand off
-				final String code = extractCodeFromLatexToSympy(latexToSympyResponse);
-
-				// 3. sympy code string to amr json
-				sympyToAMRRequest = createSympyToAMRTask(code);
-				sympyToAMRResponse = taskService.runTaskSync(sympyToAMRRequest);
-
-				final JsonNode taskResponseJSON = mapper.readValue(sympyToAMRResponse.getOutput(), JsonNode.class);
-				final ObjectNode amrNode = taskResponseJSON.get("response").get("amr").deepCopy();
-				responseAMR = mapper.convertValue(amrNode, Model.class);
-			} catch (final Exception e) {
-				log.error(messages.get("task.mira.internal-error"), e);
-				throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, messages.get("task.mira.internal-error"));
-			}
-		} else {
-			try {
-				// Create a request for SKEMA with the cleaned-up equations.
-				final JsonNode skemaRequest = mapper.createObjectNode().put("model", "petrinet").set("equations", equationsReq);
```

Comment:
Reminder we need a corresponding change on the UI side of things to the skema option off.

Reply:
Coming in a PR really soon!